src/components/MultiHopTrade/components/TradeConfirm/hooks/useTradeButtonProps.tsx

```diff
@@ -61,10 +65,36 @@ export const useTradeButtonProps = ({
     activeTradeId,
   })
 
+  const sellAccountId = useAppSelector(selectFirstHopSellAccountId)
+
   const handleTradeConfirm = useCallback(() => {
     if (!activeQuote) return
+
+    const firstStep = activeQuote.steps[0]
+    const lastStep = activeQuote.steps[activeQuote.steps.length - 1]
+    const swap: Swap = {
+      id: uuidv4(),
+      createdAt: Date.now(),
+      updatedAt: Date.now(),
+      sellAccountId,
+      swapperName: activeQuote.swapperName,
+      sellAsset: firstStep.sellAsset,
+      buyAsset: lastStep.buyAsset,
+      sellAmountCryptoBaseUnit: firstStep.sellAmountIncludingProtocolFeesCryptoBaseUnit,
+      buyAmountCryptoBaseUnit: lastStep.buyAmountAfterFeesCryptoBaseUnit,
+      metadata: {
+        lifiRoute: activeQuote.steps[0]?.lifiSpecific?.lifiRoute,
+        chainflipSwapId: firstStep?.chainflipSpecific?.chainflipSwapId,
+        stepIndex: currentHopIndex,
+        relayTransactionMetadata: firstStep?.relayTransactionMetadata,
```

Comment:
q: any reason `lifiRoute` uses `activeQuote` instead of `firstStep` for other metadata?